forward/fwd.go

```diff
@@ -130,7 +141,7 @@ type UrlForwardingStateListener func(*url.URL, int)
 // New creates an instance of Forwarder based on the provided list of configuration options
 func New(setters ...optSetter) (*Forwarder, error) {
 	f := &Forwarder{
-		httpForwarder:  &httpForwarder{flushInterval: time.Duration(100) * time.Millisecond},
+		httpForwarder:  &httpForwarder{flushInterval: time.Duration(100) * time.Millisecond, log: log.New()},
```

Comment:
I would prefer this to be: log.StandardLogger()

Which is where all direct logging calls to the log package get routed to. That will preserve any behavior where someone has modified the StandardLogger globally and expects the current behavior.

Reply:
Yes, that sounds good!  Incidentally, that would solve some of our problems, too. :)